### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by vidyavemulapalli**
Added CLI Installation guide steps to README.md

### CODE REVIEW

**Comment by vidyavemulapalli** [comment]
@vasu31dev Resolved the merge conflicts. it is now good for review. Thanks

File: README.md [hunk 1]
```diff
@@ -91,11 +91,41 @@ Before you begin, there are some essential requirements you must meet. Please re
...
+2. Create a Playwright Test Directory
+   ```bash
+    mkdir playwright-e2e-tests
+    cd playwright-e2e-tests
+   ```
+3. Install vasu-playwright-cli
+   ```bash
+   npm i -D vasu-playwright-cli
+   ```
+   This step installs all the dependencies, libraries and playwright browsers and other features automatically that are needed for the project.
```
Comment (vasu31dev) [inline README.md:114]: linting, husky and other features can be added @vidyavemulapalli

File: README.md [hunk 2]
```diff
@@ -91,11 +91,41 @@ Before you begin, there are some essential requirements you must meet. Please re
...
+
+Please refer to the installation guide [here](https://github.com/vasu31dev/playwright-ts-cli#installation) for more information and how to intialise the project.
+
+#### Manual Installation
+
+This is a step by step prcess and please refer to the [Installation section](docs/Installation.md#installation) for complete instructions on setting up the project on your local machine.
```
Comment (vasu31dev) [inline README.md:120]: What is the difference between CLI and Manual installation

**Comment by vidyavemulapalli** [comment]
Addressed the review comments. Please review the latest changes, thanks

**Review by vasu31dev — APPROVED** [review]
Looks good

**Pull request merged**